### ISSUE TITLE
[BUG] search dropdown is hidden on smaller screens

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#7003]
### Describe the bug

see screenshot

The problem is in vanilla/base 

```
.tc-block-dropdown-wrapper {
	position: relative;
}
```

If commented out `position: relative` it seems to work. Since this line is part of the vanilla/base since 2015 and no changes have been made, it's not clear if there are any side effects. 

I didn't find any side effects if it is commented out like this: 

```
.tc-block-dropdown-wrapper {
<!--	position: relative; -->
}
```

### To Reproduce

- Go to tiddlywiki.com
- make the browser window small -> so sidebar is on top
- search for "test"
- see the problem

### Screenshots

![image](https://user-images.githubusercontent.com/374655/197000986-0c79876a-fca5-4293-b30f-b7edd9c17112.png)

### TiddlyWiki Configuration

tiddlywiki.com v5.2.3

**Comment by pmario** [comment]
Did some testing. It does **not** interfere with https://tiddlywiki.com/#Customising%20search%20results

**Comment by Jermolene** [comment]
Hi @pmario it's great that removing the `position: relative` fixes the overlap issue, but I am worried that it cannot be backwards compatible to globally change `tc-block-dropdown-wrapper`. I would feel safer adding a new class to the search results dropdown that overrides the `position` CSS property.

**Comment by Alzacon** [comment]
I think the dropdown is hidden because it is inside the sidebar and this last has `overflow: auto;`
```
<div class="tc-sidebar-scrollable" style="overflow: auto;">
```
I don't find the origin of this inline style. The scrolable widget create other styles and they appears in the CSS class.

Its class `tc-sidebar-scrollable` is used in a media query (+960px) and it also have properties for the overflow, that aren't used in the desktop UI because there is this inline style.

If you disable this inline style in the browser and it also works.

**Comment by FlashSystems** [comment]
Sorry for my late response. @pmario noticed me of this PR to check if it collides with PR #6887. While adding this change to the demo for PR #6887 to check for problems, I noticed that the proposed change is not correct. To comment out the style in `base.tid` a CSS comment should be used. I think the correct change should be:

```css
.tc-block-dropdown-wrapper {
	/* position: relative; */
}
```

I've added the change to the [demo](https://flashsystems.github.io/FieldTemplatesDemo/abspopup.html) for PR #6887 if anyone wants to verify it for themselves.
